### PR TITLE
Export/ Import golinks

### PR DESCRIPTION
**Pull request by dndquynh**
Export: export existing golinks to a .txt file
Import: import new golinks/ overwrite existing golinks

- Export

download-golinks

- Import

upload-golinks

### CODE REVIEW

**Review by hieupham7127 — COMMENTED** [review]
spacing between icon and the title was a bit off

**Review by hieupham7127 — CHANGES_REQUESTED** [review]
can you change the file extension from .txt to .json?

File: popup/popup.js [hunk 1]
```diff
@@ -116,3 +118,57 @@ $(collapseBtn).on("click", function() {
...
+                data[entry[0]] = entry[1].url;
+            });
+            save(data);
+        })
+    } catch (e) {
+        alert("Failed! Error: " + e.toString());
+    }
+};
+
+function save(data) {
```
Comment (hieupham7127) [inline popup/popup.js:138]: Avoid naming function with too little wording. What are you trying to save here?

**Review by hieupham7127 — COMMENTED** [review]
everything else looks good

File: popup/popup.js (hunk 1 above)
Comment (dndquynh) [reply popup/popup.js]: Right, I'm trying to save all the golinks. I just changed the function name to exportGoLinks.

**Comment by dndquynh** [comment]
> can you change the file extension from .txt to .json?

Done

**Comment by dndquynh** [comment]
> spacing between icon and the title was a bit off

Added some spacing, does this look good enough?
Screen Shot 2021-05-13 at 9 20 01 PM

**Review by hieupham7127 — APPROVED** [review]

**Pull request merged**